packages/node/src/authenticatedFetch/fetchFactory.ts

```diff
@@ -22,33 +22,89 @@
 import { JWK, JWT } from "jose";
 import { fetch } from "cross-fetch";
 import { v4 } from "uuid";
+import { ITokenRefresher } from "../login/oidc/refresh/TokenRefresher";
 
 // node-fetch fetch has an additional property (isRedirect) which prevents using
```

Comment:
This comment is a bit confusing - I'd suggest this instead, and explain why we want to use `typeof fetch`:
```suggestion
// We don't import the node-fetch fetch, as it has an additional property (isRedirect) that would prevent us from using `typeof fetch`, which we want to use because...????
```

Reply:
Actually, the comment is outdated, and `fetchType` is no longer necessary since I switched from `node-fetch` to `cross-fetch`, the latter providing type definitions aligned with the ones we expect. Resolved in c122bd2c2b0c73842f1c35ddc853f16d12b67201